src/components/ResultCard/ResultCard.jsx

```diff
@@ -3,15 +3,15 @@ import './ResultCard.css';
 
 const ResultCard = (movie) => {
   const { title, poster_path: posterPath } = movie;
-  const linkToFicheFilm = (e) => {
+  const linkToMovieProfile = (e) => {
     e.preventDefault();
   };
   return (
     <div
-      className="resultcard"
+      className="resultCard"
       role="button"
-      onClick={linkToFicheFilm}
-      onKeyPress={linkToFicheFilm}
+      onClick={linkToMovieProfile}
+      onKeyPress={linkToMovieProfile}
```

Comment:
So, this component (`div`)... 
- Must do nothing when we click on it ? 
- And you wrote 2 EListeners to ensure this ?
- And it's calling a "LinkBlabla" function, but there is absolutely no `<a>` nor `<Link>` anywhere in sight ?
- As long as I'm here : why don't you use a `button` ?

Reply:
For now its dont have any purpose than set the comportement, its set out for link to another components/page when we will set up the route soon. And for the Key listener thanks eslint Who have contrain me to ad it for validation. No button cause We dont wan't an ugly one on our cardResult especialy on mobile version.